recipes/rmats2sashimiplot/meta.yaml

```diff
@@ -20,5 +20,16 @@ build:
   noarch: python
```

Comment:
The current build failed for python3. Can you add a `skip: True # [not py27]`? Subsequently, `python ==2.6*|2.7*` can be replaced by `python`.